src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -18,235 +27,406 @@ export const AboutApp: React.FC = () => {
-            about, and collect art.
+            Artsy envisions a future where everyone is moved by art every day.
+            To get there, we’re expanding the art market to support more artists
+            and art around the world. As the leading marketplace to discover,
+            buy, and sell fine art, Artsy believes that the process of buying
+            art should be as brilliant as art itself. That’s why we’re dedicated
+            to making a joyful, welcoming experience that connects collectors
+            with the artists and artworks they love.
           </Text>
+        </Column>
+      </GridColumns>
+      <Spacer mt={6} />
+      <Rail
+        title="Discover Artworks Just for You"
+        subTitle="On Artsy"
+        getItems={() => {
+          const items: RailComponentProps[] = [
+            {
+              imgUrl:
+                "https://d7hftxdivxxvm.cloudfront.net/?resize_to=fit&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FoOhfzetWDyPIdRYLVRBy8A%2Flarge.jpg&width=890&height=1046&quality=80",
+              name: "Caleb Hahne",
+              artworkTitle: "Carry Me Home, 2019",
+              gallery: "1969 Gallery",
+              width: 340,
+              height: 500,
+            },
+            {
+              imgUrl:
+                "https://d7hftxdivxxvm.cloudfront.net/?resize_to=fit&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FSbbu7kX5gzxEUfL4Jwyk6Q%2Flarge.jpg&width=890&height=1112&quality=80",
+              name: "Amy Sherald",
+              artworkTitle: "The Make Believer (Monet's Garden), 2016",
+              gallery: "The Studio Museum in Harlem",
+              width: 300,
+              height: 375,
+            },
+            {
+              imgUrl:
+                "https://d7hftxdivxxvm.cloudfront.net/?resize_to=fill&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FE_tdYYu4XOeuVK8ZHFwJSw%2Flarge.jpg&width=890&height=890&quality=80",
+              name: "Frank Stella",
+              artworkTitle:
+                "Sinjerli Variations Squared with Colored Grounds VI, 1981",
+              gallery: "Heritage Auctions",
+              width: 300,
+              height: 300,
+            },
+            {
+              imgUrl:
+                "https://d7hftxdivxxvm.cloudfront.net/?resize_to=fit&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2Fm4kmYLBKSkExMXlxjQ4i8A%2Flarge.jpg&width=890&height=1046&quality=80",
+              name: "Inka Essenhigh",
+              artworkTitle: "Old Trees in Fall, 2018",
+              gallery: "Miles McEnery Gallery",
+              width: 340,
+              height: 500,
+            },
+          ]
+
+          return items.map(item => {
+            return <RailComponent {...item} />
+          })
+        }}
+      />
+
+      <SellWithArtsySection />
+      <Spacer mt={4} />
+      <AppDownloadSection />
+      <CollectorInfoSection />
+      <ArtsySpecialistsSection />
```

Comment:
In lieu of the outer margins on all these components you'd control spacing here with spacers.

Reply:
Ah jinx 😄 - commented on that here: https://github.com/artsy/force/pull/9313#discussion_r803127183